qiskit/dagcircuit/dagdependency.py

```diff
@@ -164,8 +164,8 @@ def to_networkx(self):
                 dag_networkx.add_edge(self.get_node(source_id), self.get_node(dest_id), **edge)
         return dag_networkx
 
-    def to_retworkx(self):
-        """Returns the DAGDependency in retworkx format."""
+    def to_reseaux(self):
+        """Returns the DAGDependency in reseaux format."""
```

Comment:
We can't change method names like this - we'll need to go through a deprecation and backwards-compatibility period.

Reply:
I am working towards making them the same object, but that will depends on how we release 0.12